compiler/rawdata2hdf5/rawdata2hdf5

```diff
@@ -99,17 +105,21 @@ def create_hdf5(data_list, output_path):
 
 
 def main():
-    parser = get_parser()
-
-    args = parser.parse_args()
+    try:
+        parser = get_parser()
+        args = parser.parse_args()
+        verify_args(parser, args)
+        create_hdf5(args.data_list, args.output_path)
+        return 0
 
-    verify_args(parser, args)
+    except UnicodeDecodeError:
+        print("UnicodeDecode Error. data_list may not be the text file.")
+        return 1
 
-    create_hdf5(args.data_list, args.output_path)
+    except Exception as e:
+        print(e)
+        return 1
 
 
 if __name__ == '__main__':
-    try:
-        main()
-    except UnicodeDecodeError:
-        raise SystemExit("UnicodeDecode Error. data_list may not be the text file.")
+    sys.exit(main())
```

Comment:
I'm not sure but this try-catch statement is necessary? If you raise the error in the function, it will be exit with desired return value.

Reply:
> If you raise the error in the function, it will be exit with desired return value.

At first I wrote the code as you said, but the code did not work correctly during testing.

The test codes save logs as follows.

```
rawdata2hdf5 \
--data_list ${inputfile} \
--output_path ${outputfile} > ${filename}.log
```

When I raised an error in the function, the log was not written to `${filename}.log` (don't know why..). The logs were correctly written to the file when I use the function `print`.

Reply:
@jinevening Please add `2>&1` at the end of the command.
```bash
$ rawdata2hdf5 \
--data_list ${inputfile} \
--output_path ${outputfile} > ${filename}.log 2>&1
```

Reply:
> @jinevening Please add 2>&1 at the end of the command.

@mhs4670go  Thanks for the suggestion. But is it ok to use unhandled exception?

In cpp program (circle-quantizer), the release test team did not accept unhandled exceptions.

If unhandled exception is ok in Python program, I'll remove try-catch statement.

The log looks as below.

```
hjj@hjj-ubuntu:~/ONE.2/install/test$ ../bin/rawdata2hdf5 -l wronglist.txt -o out.h5
Traceback (most recent call last):
  File "../bin/rawdata2hdf5", line 118, in <module>
    main()
  File "../bin/rawdata2hdf5", line 115, in main
    create_hdf5(args.data_list, args.output_path)
  File "../bin/rawdata2hdf5", line 88, in create_hdf5
    raise Exception("No such file. " + filename)
Exception: No such file. **non-existing-file.data**
```

CC  @seanshpark